index.html

```diff
@@ -0,0 +1,499 @@
+              <p class="choose-plan__text">Unlimited Bandwitch</p>
+            </li>
+            <li class="choose-plan__list">
+              <img
+                src="/img/check-2.svg"
+                width="24"
+                height="24"
+                alt="check-2"
+              />
+              <p class="choose-plan__text">Encrypted Connection</p>
+            </li>
+            <li class="choose-plan__list">
+              <img
+                src="/img/check-2.svg"
+                width="24"
+                height="24"
+                alt="check-2"
+              />
+              <p class="choose-plan__text">No Traffic Logs</p>
+            </li>
+            <li class="choose-plan__list">
+              <img
+                src="/img/check-2.svg"
+                width="24"
+                height="24"
+                alt="check-2"
+              />
+              <p class="choose-plan__text">Works on All Devices</p>
+            </li>
+          </ul>
+          <h3 class="choose-plan__subtitle">Free</h3>
+          <button class="button choose-plan__button" type="button">
+            Select
+          </button>
+        </div>
+      </section>
+
+      <section class="global-net">
+        <h2 class="global-net__title">Huge Global Network of Fast VPN</h2>
+        <p class="global-net__subtitle">
+          See <strong class="text-accent">LaslesVPN</strong> everywhere to make
+          it easier for you when you move locations.
+        </p>
+        <img
+          src="./img/huge-global.svg"
+          width="1060"
+          height="538"
+          alt="global-lassmap"
+        />
+        <img
+          src="./img/sponsored.svg"
+          width="1136"
+          height="538"
+          alt="logo-sponsores"
+        />
+      </section>
+
+      <section class="trusted-customer">
+        <h2 class="title trusted-customer__title">
+          Trusted by Thousands of Happy Customer
+        </h2>
+        <p class="trusted-customer__subtitle">
+          These are the stories of our customers who have joined us with great
+          pleasure when using this crazy feature.
+        </p>
+        <div class="trusted-customer__slider">
+          <div class="trusted-customer__item">
+            <div class="trusted-customer--datauser">
+              <img
+                src="./img/riezh-robert.svg"
+                width="50"
+                height="50"
+                alt="riezh-robert"
+              />
+              <div class="trusted-customer__username">
+                <p class="trusted-customer__name">Viezh Robert</p>
+                <p class="trusted-customer__born">Poland</p>
+              </div>
+              <div class="trusted-customer__rating">
+                <p class="trusted-customer__text">4.5</p>
+                <img src="./img/star.svg" alt="star" width="16" height="16" />
+              </div>
+            </div>
+            <blockquote class="trusted-customer--text">
```

Comment:
`--text` is also a modifier. Please use `__text`, or if you want another class name, `__quote`, for example, would work also.

Reply:
+